static/src/javascripts/projects/common/modules/experiments/low-friction-participation.js

```diff
@@ -0,0 +1,261 @@
+define([
```

Comment:
Some stylistic comments (thought I'd pull these out of the body of the diff)

- we don't really use single-var declarations any more
- not sure if we need to define functions as vars